packages/@ourworldindata/grapher/src/scatterCharts/ScatterPlotChart.tsx

```diff
@@ -564,8 +564,10 @@ export class ScatterPlotChart
     }
 
     @computed get legendItems(): ColorScaleBin[] {
-        return this.colorScale.legendBins.filter((bin) =>
-            this.colorsInUse.includes(bin.color)
+        return this.colorScale.legendBins.filter(
+            (bin) =>
+                this.colorsInUse.includes(bin.color) &&
+                !bin.label?.match(/no data/i)
```

Comment:
It probably makes sense to explicitly reference `NO_DATA_LABEL` from `ColorScale.ts` here, which also makes it easier for someone to follow the link back to where that bin is created.

Reply:
Ah, that's much better. It was definitely feeling wrong to compare against a copy the phrasing we happen to be using for the moment…